app/components/candidate_interface/gcse_qualification_review_component.rb

```diff
@@ -72,6 +72,9 @@ def present_grades
           "#{grades['chemistry']} (Chemistry)",
           "#{grades['physics']} (Physics)",
         ]
+      elsif application_qualification.subject == 'english' && application_qualification.grades
```

Comment:
1. I switched the English GCSE flag on and updated my GCSE grades. 
2. I then turned the flag off and edited my English GCSE grade. 
3. When I went to the review component it was still showing the multiple grades from 1.

Reply:
<img width="972" alt="review_component" src="https://user-images.githubusercontent.com/5256922/99401964-b76b9b00-28e0-11eb-822d-6e7f282e787f.png">

Reply:
Thanks for spotting this Toby. This has been fixed by setting `grades` to `nil` when we set `grade`. 

I have just tested it again on the heroku app to verify that it is now working as expected.

Reply:
Yea this is a variation of the same bug i found earlier 👍